servers/rendering_server.cpp

```diff
@@ -2286,6 +2290,14 @@ RID RenderingServer::instance_create2(RID p_base, RID p_scenario) {
 	return instance;
 }
 
+bool RenderingServer::is_render_loop_enabled() const {
+	return render_loop_enabled;
+}
+
+void RenderingServer::set_render_loop_enabled(bool render_loop_enabled) {
+	this->render_loop_enabled = render_loop_enabled;
```

Comment:
The parameter should be `p_enabled` (and then you don't need `this->`).

Reply:
Fixed *(old Java quirks :stuck_out_tongue:)*